doc/manipulating_images/manipulating_images.rst

```diff
@@ -375,32 +375,30 @@ of the transformation matrix (i.e., affine).
 Image operations: creating a ROI mask manually
 ===============================================
 
-This section shows manual steps to create and further modify a ROI
-(region of interest) spatial mask. They represent a means for "data folding",
-that is, extracting and later analyzing data from a subset of voxels rather
-than the entire brain images. As a convenient side effect, this can help
-alleviate the curse of dimensionality (i.e., statistical problems that
-arise in the context of high-dimensional input variables).
```

Comment:
I don't understand why those lines were added to the narrative documentation. My idea when posing the issue was to move all this section of the narrative documentation to an example.